src/user/userController.js

```diff
@@ -0,0 +1,12 @@
+const User = require("./userModel");
+
+exports.addUser = async (req, res) => {
+    try {
+      const newUser = new User(req.body);
+      await newUser.save();
+      res.status(200).send({ message: `Successfully added ${newUser.username}`, newUser });
```

Comment:
we might not need to return the newUser in response, it is useful for testing, but might not be needed in the actual app, a message might be enough but let's leave it for now